src/Caching/Test/Microsoft.Python.Analysis.Caching.Tests.csproj

```diff
@@ -0,0 +1,43 @@
+﻿<Project>
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <RootNamespace>Microsoft.Python.Analysis.Caching.Tests</RootNamespace>
+    <AssemblyName>Microsoft.Python.Analysis.Caching.Tests</AssemblyName>
+  </PropertyGroup>
+  <PropertyGroup>
+    <!-- 
+      1701, 1702 - "You may need to supply assembly policy"
+    -->
+    <NoWarn>1701;1702$(NoWarn)</NoWarn>
+    <LangVersion>7.2</LangVersion>
+  </PropertyGroup>
+  <Import Project="..\..\..\build\NetStandard.settings" />
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  <ItemGroup>
+    <None Remove="app.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.0" />
+    <PackageReference Include="Microsoft.Win32.Registry" Version="4.5.0" />
```

Comment:
Registry?

Reply:
We actually have it in all test assemblies. Might be that test framework is bringing it in?

Reply:
Maybe a copy/paste error.

Reply:
Removed.